plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -1814,6 +1829,35 @@ GraphHandlePtr QtOSGViewer::drawtrimesh(const float* ppoints, int stride, const
     return GraphHandlePtr(new PrivateGraphHandle(shared_viewer(), handle));
 }
 
+class GlobalLOD : public osg::LOD
+{
+public:
+    void traverse(osg::NodeVisitor& nv)
+    {
+        osg::CullStack *cs;
+        if (nv.getTraversalMode() == osg::NodeVisitor::TRAVERSE_ACTIVE_CHILDREN && _rangeMode==DISTANCE_FROM_EYE_POINT) {
+            float required_range = nv.getDistanceToViewPoint(getCenter(),true);
+            if ((cs = dynamic_cast<osg::CullStack *>(&nv))) {
```

Comment:
To be more readable, I think first do cs = dynamic_cast ... and then check if cs is not null: if(cs)

plus, use more meaningfull variable names please. cs => cullStack
;)